data/net/minecraft/util/Crypt.mapping

```diff
@@ -34,8 +36,26 @@ CLASS net/minecraft/util/Crypt
 		COMMENT Creates a Cipher instance using the AES/CFB8/NoPadding algorithm. Used for protocol encryption.
 		ARG 0 opMode
 		ARG 1 key
+	METHOD rsaPublicKeyToString (Ljava/security/PublicKey;)Ljava/lang/String;
+		ARG 0 key
+	METHOD rsaStringToKey (Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;Lnet/minecraft/util/Crypt$ByteArrayToKeyFunction;)Ljava/security/Key;
+		ARG 0 keyBase64
+		ARG 3 keyFunction
 	METHOD setupCipher (ILjava/lang/String;Ljava/security/Key;)Ljavax/crypto/Cipher;
 		COMMENT Creates the Cipher Instance.
 		ARG 0 opMode
 		ARG 1 transformation
 		ARG 2 key
+	METHOD stringToPemRsaPrivateKey (Ljava/lang/String;)Ljava/security/PrivateKey;
+		ARG 0 keyBase64
+	METHOD stringToRsaPublicKey (Ljava/lang/String;)Ljava/security/PublicKey;
+		ARG 0 keyBase64
+	CLASS SaltSignaturePair
+		METHOD <init> (Lnet/minecraft/network/FriendlyByteBuf;)V
+			ARG 1 buf
+		METHOD write (Lnet/minecraft/network/FriendlyByteBuf;Lnet/minecraft/util/Crypt$SaltSignaturePair;)V
+			ARG 0 buf
```

Comment:
`buf` -> `buffer`, imo, as per [Standards](https://parchmentmc.org/docs/mappings) Mappings 4. "Avoid abbreviations or acronyms."

A cursory search through the repo indicates we have in multiple places, both `buffer` and `buf` used already for `FriendlyByteBuf` parameters.

Reply:
Is `buf` not valid even if it's already part of the classname?

Reply:
imo, no, that doesn't pass the smell test as to if we should use the abbreviation/class name vs. prefer the full word. Which is consistent with past sentiment on the matter, i.e. [here](https://github.com/ParchmentMC/Parchment/pull/24#issuecomment-868975548) where we standardized on `buffer` (vs. `buf` or `byteBuf`)